cluster/config-defaults.yaml

```diff
@@ -33,7 +33,7 @@ skipper_ingress_memory: "1Gi"
 skipper_ingress_tracing_buffer: "8192"
 
 # skipper default filters
-skipper_default_filters: 'enableAccessLog(4,5) -> lifo(2000,20000,"10s")'
+skipper_default_filters: 'enableAccessLog(4,5) -> lifo(2000,20000,"3s")'
```

Comment:
@szuecs does this mean that skipper drops requests after 3s regardless of the stack size? If yes, I think we should first check what requests will fail as it imposes a hidden latency limit for every service running on K8s.

Reply:
@sarslanhan no it means only that skipper is not able to pop from stack within 3s.
10s would be also bad for long blocking calls, so don't worry with that.